pkg/devicemapper/devmapper_wrapper_dynamic.go

```diff
@@ -1,6 +0,0 @@
-// +build linux,cgo,!static_build
-
-package devicemapper
-
-// #cgo pkg-config: devmapper
-import "C"
```

Comment:
If you keep this file, you won't have to add `#cgo pkg-config: devmapper` to 3 other places. I guess the original reason for removing it was that you needed `pkg-config: --static devmapper` but it is no longer the case.